src/structs.jl

```diff
@@ -100,6 +100,19 @@ end
 """
 $(TYPEDEF)
 
```

Comment:
Maybe add a short description of the type, besides the automatic stuff?

Reply:
very true, added below